helm/hookshot/values.yaml

```diff
@@ -129,64 +129,64 @@ hookshot:
     github:
```

Comment:
I think also comment out these top level things too

Reply:
Happy with that, although testing showed it seems to work with them defined but not :smile: 

What about [widgets](https://github.com/matrix-org/matrix-hookshot/blob/main/helm/hookshot/values.yaml#L246) while we're at it?

Ideally, but no showstopped, AFAIU its best practice to use an empty mapping such as:
```
    queue: {}
```

However I tested this and it caused the same behaviour (looking for localhost) so I suspect it's something in the code that falls back to that?

Reply:
Queue should *just* contain `monolithic: true`, rather than empty object.

Reply:
> Queue should _just_ contain `monolithic: true`, rather than empty object.

As discussed we're leaving this commented out completely.